packages/backend/src/config.mjs

```diff
@@ -1,16 +1,57 @@
-import { ethers } from 'ethers'
 import { config } from 'dotenv'
+import { ethers } from 'ethers'
+import pkg from 'hardhat'
+const { ethers: hardhatEthers } = pkg
 config()
 
-export let UNIREP_ADDRESS = '0x4D137bb44553d55AE6B28B5391c6f537b06C9cc3'
-export const ETH_PROVIDER_URL =
-  process.env.ETH_PROVIDER_URL ??
-  'https://arb-goerli.g.alchemy.com/v2/hfFfXlX8rR8YvrALiJ8b7ZtIPRGY1GTM'
-// export const ETH_PROVIDER_URL =
-//   process.env.ETH_PROVIDER_URL ?? 'http://127.0.0.1:8545'
-
 export const DB_PATH = process.env.DB_PATH ?? ':memory:'
 
-export const provider = ETH_PROVIDER_URL.startsWith('http')
-  ? new ethers.providers.JsonRpcProvider(ETH_PROVIDER_URL)
-  : new ethers.providers.WebSocketProvider(ETH_PROVIDER_URL)
+export const NETWORK = {
+  arbitrumGoerli: {
+    name: 'arbitrum-goerli',
+    provider: new ethers.providers.JsonRpcProvider(
+      `https://arbitrum-goerli.infura.io/v3/${process.env.INFURA_KEY}`
+    ),
+    explorer: 'https://goerli.arbiscan.io',
+    unirepAddress: '0x4D137bb44553d55AE6B28B5391c6f537b06C9cc3',
```

Comment:
I think it might not be the same address in different network
now we rely on [global-factory](https://github.com/vimwitch/global-factory) to deploy a smart contract
if we are not using it in the future it could be different address